mw-webapp/src/service/auth/logIn.ts

```diff
@@ -1,6 +1,9 @@
 import {signInWithRedirect} from "firebase/auth";
 import {auth, provider} from "src/firebase";
 
+/**
+ * Log in
```

Comment:
Oh, before this comment I struggled using thisfunction, but now I feel real transparency, thanks ;)

Reply:
Changed